package.json

```
@@ -47,5 +47,10 @@
     "typescript": "^4.0.5"
   },
   "author": "wwwy3y3",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/keycloak/keycloak-nodejs-admin-client.git"
+  },
+  "homepage": "http://keycloak.org"
```

This URL redirects, so let's use the final URL here.
```suggestion
  "homepage": "https://www.keycloak.org/"
```